readme.txt

```diff
@@ -122,6 +122,10 @@ To see a list of all the premium extensions and learn more visit [http://wiredim
 
 == Changelog ==
 
+= 1.1.12 =
+* Update the way the Help tab is removed for Board Members due to the 'contextual_help' filter being deprecated.
```

Comment:
Do you want to change the first word to the past tense "Updated" since that's how we've _mostly_ done it previously? It would be nice to shoot for consistency.